### DISCUSSION TITLE
.bin file damaged or simply not recognized?

### DISCUSSION THREAD

**Discussion by DerpingDerp** [Johnson070/MiBand-5-watchface-editor#24] [General]
Hi, Vladmir.
I've found your aswesome GUI editor and would like to donate your job, but your Paypal is set up to company-only donations, or something like that :) I can't donate you because **Paypal says you don't accept private** donations.

But, going straight to my issue.
I've made quite nice watchface for MiBand 6 thanks to you, but file that is compiled - .bin - can't be opened by near all MiBand-supporting apps, like Notify for Miband or even MiFit itselt. I tried to upload my .bin to the phone, local folder for local watchfaces. I can see it in MiFit but **synchronization takes forever**. It's **below maximum weight - 1,2 MB**, so it should fit. Also it works fine in your GUI editor.

I've spent 4 days making this watchface and well, I can't use it. Vid about your program, my watchface and tutorial how to use it (with credits to you) is on the way, but sadly I can't go through this synch issue. All other downloaded .bins are working fine without any issue, but that one made in your GUI cause the problem.

If there's any way to contact you, let me know - through Discord, Amazfit forums etc ;)

Here's my small gem I made.

![BrodatyWatchface-PL-version_packed_animated](https://user-images.githubusercontent.com/84633370/119232059-0f02f180-bb24-11eb-8e33-b8479497d35c.gif)

**Comment by DerpingDerp** [comment]
Ok. So I found out that I've to enable palette mode, after that I'm able to upload watchface.
However steps counter, calories counter, PAI number and pulse isn't working. In GUI it's visable, it's working, I can see it increases or decreases while using sliders. But it's not working after uploading to MiBand.
LOCK ON is misplaced. ON is OFF and OFF is ON ;)

Another thing GUI have issues with deleted images. Sometimes I've to use previous one, so GUI knows that I'm "using" deleted ones ;)

Vladimir, if you need to see this issue "live", for example through Discord, let me know! :)

**Reply by Johnson070** [reply]
Ok)
I can tomorrow or today, but in the evening.  Johnson070#5729

**Reply by Johnson070** [reply]
Glad you liked my program)

**Comment by hiozcan** [comment]
Hello there.
I use translation because my English is not good.
How can I upload my screen design that I designed on my Mi band 5 bracelet?
I cannot install on the device even though I copied it to the "Galaxy S20\Phone\Android\data\com.xiaomi.hm.health\files\watch_skin_file" file path.

Can you help me please.
